test.js

```diff
@@ -197,6 +197,17 @@ test('SR-Flip-Flip : No Change', t => {
   clock.switchOff()
 })
 
+test('T-Flip-Flip: 1', t => {
+  const T = wires(1)
+  const qqbar = wires(2)
+  const clock = new Pulse(500, 1)
+  clock.switchOn()
+  const ff = new TFlipFlop(T, qqbar, clock)
+  const ioHandler = new StringIO(ff)
+  t.is(ioHandler.input('1'), '1')
```

Comment:
ioHandler returns an empty string.

Also, as this is a T-Flip Flop, for a input `1` the output should be `0`.